test/AspNetCoreModule.Test/aspnetcoremodule.test.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net452</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>AspNetCoreModule.Test</AssemblyName>
```

Comment:
You can remove `AssemblyName` and `PackageId` from this file. These settings are redundant